examples/reference-entity-model/entities/Customer.entity.json

```diff
@@ -1,98 +1,111 @@
-{
-  "info": {
-    "title": "Customer",
-    "version": "0.0.1",
-    "baseUri": "http://example.org/"
-  },
-  "definitions": {
-    "Customer": {
-      "required": [
-        "name"
-      ],
-      "primaryKey": "customerId",
-      "properties": {
-        "customerId": {
-          "datatype": "integer"
-        },
-        "name": {
-          "datatype": "string",
-          "collation": "http://marklogic.com/collation/codepoint"
-        },
-        "nicknames": {
-          "datatype": "array",
-          "description": "Example of a multi-value property of simple values",
-          "items": {
-            "datatype": "string"
-          }
-        },
-        "shipping": {
-          "datatype": "array",
-          "description": "Example of a multi-value property of structured values",
-          "items": {
-            "$ref": "#/definitions/Address"
-          }
-        },
-        "billing": {
-          "description": "Example of a single-value structured property",
-          "$ref": "#/definitions/Address"
-        },
-        "birthDate": {
-          "datatype": "date"
-        },
-        "status": {
-          "datatype": "string"
-        },
-        "customerSince": {
-          "datatype": "date"
-        },
-        "orders": {
-          "datatype": "array",
-          "description": "Example of a relationship to another entity type",
-          "items": {
-            "$ref": "http://example.org/Order-0.0.1/Order"
-          }
-        }
-      }
-    },
-    "Address": {
-      "required": [ ],
-      "pii": [ ],
-      "elementRangeIndex": [ ],
-      "rangeIndex": [ ],
-      "wordLexicon": [ ],
-      "properties": {
-        "street": {
-          "datatype": "array",
-          "items": {
-            "datatype": "string",
-            "collation": "http://marklogic.com/collation/codepoint"
-          }
-        },
-        "city": {
-          "datatype": "string",
-          "collation": "http://marklogic.com/collation/codepoint"
-        },
-        "state": {
-          "datatype": "string",
-          "collation": "http://marklogic.com/collation/codepoint"
-        },
-        "zip": {
-          "$ref": "#/definitions/Zip"
-        }
-      }
-    },
-    "Zip": {
-      "required": [ ],
-      "properties": {
-        "fiveDigit": {
-          "datatype": "string",
-          "collation": "http://marklogic.com/collation/codepoint"
-        },
-        "plusFour": {
-          "datatype": "string",
-          "collation": "http://marklogic.com/collation/codepoint"
-        }
-      }
-    }
-  }
-}
+{
+  "info": {
+    "title": "Customer",
+    "version": "0.0.1",
+    "baseUri": "http://example.org/"
+  },
+  "definitions": {
+    "Customer": {
+      "required": [
+        "name"
+      ],
+      "primaryKey": "customerId",
+      "properties": {
+        "customerId": {
+          "datatype": "integer",
+          "facet": true

```

Comment:
Let's make "status" a facet instead of customerId - a user would likely never have a facet on a unique value like this.